doc/sphinx/changes.rst

```diff
@@ -455,7 +455,7 @@ Tactic language
   (`#11882 <https://github.com/coq/coq/pull/11882>`_,
   by Hugo Herbelin).
 - **Added:**
-  Ltac2 notations for reductions in terms: :n:`eval @red_expr in @ltac2_term`
+  Ltac2 notations for reductions in terms: :n:`eval @red_expr in @constr`
```

Comment:
Where is this `constr` nonterminal defined?

Reply:
`constr` is before `doc_grammar` maps it to `term`.  I'll make it `term`.